airbyte-ci/connectors/live-tests/src/live_tests/regression_tests/conftest.py

```diff
@@ -118,7 +116,8 @@ def pytest_configure(config: Config) -> None:
     dagger_log_path.touch()
     config.stash[stash_keys.DAGGER_LOG_PATH] = dagger_log_path
     config.stash[stash_keys.PR_URL] = get_option_or_fail(config, "--pr-url")
-    config.stash[stash_keys.AUTO_SELECT_CONNECTION] = config.getoption("--auto-select-connection")
```

Comment:
Since we should only have one of auto-select or connection ID, we don't really need both flags and it creates more surface area for bugs or unexpected behavior. For example, when I passed in a connection ID via gha I couldn't find a way to turn off auto-select since pytest doesn't parse falsey values. I could have changed auto-select to default to false, but then we'd still end up in a situation where users could provide inputs that don't make sense when combined. So I decided to remove the degree of freedom.